.travis.yml

```diff
@@ -15,13 +15,13 @@ install:
   - git clone --depth=1 --branch=$GFW_BRANCH https://github.com/terasolunaorg/terasoluna-gfw.git terasolunaorg/terasoluna-gfw
   - cd terasolunaorg/terasoluna-gfw
   - sh ./mvn-build-all.sh install -Dmaven.test.skip=true -Dmaven.javadoc.skip=true -Dsource.skip=true
+  - export CARGO_DAEMON_WEBAPP_VERSION=`grep "<cargo-maven2-plugin.version>" terasoluna-gfw-parent/pom.xml | sed "s/.*<.*>\(.*\)<\/.*>.*/\1/g"`
```

Comment:
@kazuki43zoo 
Alternatively, I think you can retrieve it using Maven Help Plugin as follows. Which is better?
```console
export CARGO_DAEMON_WEBAPP_VERSION=`mvn -B -f terasoluna-gfw-parent/pom.xml help:evaluate -Dexpression=cargo-maven2-plugin.version | grep -v '^\['`
```